### PR TITLE
fix(activity): Set Info.isLocal correctly

### PR DESCRIPTION
**Pull request by lorensr**
## What was changed

- Set isLocal correctly. Was broken, reported here: https://temporalio.slack.com/archives/C01DKSMU94L/p1655837399070479
- Share workflow bundle in local activity tests to avoid an error about out of wasm memory (swc?)

### CODE REVIEW

File: packages/test/src/test-local-activities.ts [hunk 1]
```diff
@@ -12,28 +12,36 @@ import * as workflows from './workflows/local-activity-testers';
...
+  getWorker: () => Promise<Worker>;
 }
 
 const test = anyTest as TestInterface<Context>;
 
+let workflowBundle: WorkflowBundleWithSourceMap;
+
+test.before(async () => {
+  workflowBundle = await bundleWorkflowCode({ workflowsPath: require.resolve('./workflows/local-activity-testers') });
+});
```
Comment (bergundy) [inline packages/test/src/test-local-activities.ts:24]: Let's do this cleanly and put it in the test context.

**Review by bergundy — APPROVED** [review]
Nice, just one comment

File: packages/test/src/test-local-activities.ts [hunk 2]
```diff
@@ -10,30 +10,40 @@ import { RUN_INTEGRATION_TESTS } from './helpers';
...
   client: WorkflowClient;
+  getWorker: () => Promise<Worker>;
 }
 
 const test = anyTest as TestInterface<Context>;
 
+test.before(async (t) => {
+  t.context = {
+    workflowBundle: await bundleWorkflowCode({ workflowsPath: require.resolve('./workflows/local-activity-testers') }),
+  } as Context;
```
Comment (lorensr) [inline packages/test/src/test-local-activities.ts:24]: @bergundy Like this? (Without the cast, the other props are optional, which adds a lot of `?.` to the tests.)

Comment (bergundy) [reply packages/test/src/test-local-activities.ts:24]: `t.context.workflowBundle = ...`

**Review by bergundy — APPROVED** [review]

**Pull request merged**